lnwallet/reservation.go

```diff
@@ -183,6 +183,19 @@ func NewChannelReservation(capacity, fundingAmt, commitFeePerKw btcutil.Amount,
 		initiator = true
 	}
 
+	// If the total fee we'd need to pay is greater than the balance of
+	// either party, then we'll reject it as the request cannot be
+	// satisfied.
```

Comment:
Should we add a TODO here to handle the dual funder case whenever that's implemented?

Reply:
Added.